jsonrpc/eth_endpoint.go

```diff
@@ -27,11 +27,16 @@ type ethTxPoolStore interface {
 	GetPendingTx(txHash types.Hash) (*types.Transaction, bool)
 }
 
+type Account struct {
+	Balance *big.Int
```

Comment:
Why do we need narrowed Account structure?

Reply:
It is part of the conversation from yesterday on removing cross-dependencies between the packages. `jsonrpc` as it now relies on a structure targeted to work on the state transition (it even has some internal objects for merkle trie). Now, it only contains the required fields to make it work on `jsonrpc`.

Reply:
I may havent heard it. Could you give me a link to a RFC about it?

Reply:
[This](https://www.notion.so/polygontechnology/RFC-145-0b3ac0015b2047b0b655fc66c2f028fc#5aa35c955f414602a43d4f355b6db662) is the example of the implementation. It uses local objects instead of the ones from `types`. The same reasoning is applied here.